commons/src/main/java/com/powsybl/commons/reporter/Reporter.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.commons.reporter;
+
+import java.util.Map;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+public interface Reporter {
```

Comment:
Yes, sure, it should be detailed! I'll try to include that in current PR, but, even though it would help to review, maybe that could be done in a second one?

Reply:
Lack of an extensive javadoc for a beta feature is not a blocking issue for me as it might be changed later anyway but it should indeed be documented somewhere (@annetill Should we have a "beta feature" section on the website?)

Reply:
Yes, it was not clear to me what was the status of that feature. But let's not forget to document it when the main concepts are validated. As a user, I would not have been comfortable with using it in the current state !